webscrambles/src/net/gnehzr/tnoodle/server/webscrambles/ScrambleRequest.java

```diff
@@ -146,14 +147,16 @@ private static BaseFont getFontForLocale(Locale locale) {
     // This is here just to make GSON work.
     public ScrambleRequest(){}
 
-
     public String[] scrambles;
     public String[] extraScrambles = new String[0];
     public Puzzle scrambler;
     public int copies;
     public String title;
     public boolean fmc;
     public HashMap<String, Color> colorScheme;
+    public int totalAttempt;
```

Comment:
Could you add a comment explaining what values this can take on? In particular, I think you set this to -1 when it's unset? It would be nice to explain that somewhere.

Reply:
Fixed